### PR TITLE
Feat(admin-next, core-flows, link-modules, medusa, types): Inventory end to end flows

### PR DESCRIPTION
**Pull request by pKorsholm**
**What**
- Add inventory management admin-ui
- Add bulk endpoint for inventory levels

_Note: This is still missing the "reservations" part of the admin ui, that will follow when the end-to-end reservations have been added in a separate pr_

### CODE REVIEW

File: packages/core-flows/src/inventory/steps/delete-levels-by-item-and-location.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          .join(
+            ", "
+          )} because there are reserved quantities for items at locations`
+      )
+    }
+
+    const deletedIds = items.map((i) => i.id)
+    await service.softDeleteInventoryLevels(deletedIds)
+
+    return new StepResponse(void 0, deletedIds)
```
Comment (carlos-r-l-rodrigues) [inline packages/core-flows/src/inventory/steps/delete-levels-by-item-and-location.ts]:
```suggestion
    const deleted = await service.softDeleteInventoryLevels(deletedIds)

    return new StepResponse({
      ModuleRegistrationName.INVENTORY: deleted
    }, deletedIds)
```

File: packages/core-flows/src/inventory/workflows/bulk-create-delete-levels.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  creates: InventoryNext.CreateInventoryLevelInput[]
+  deletes: { inventory_item_id: string; location_id: string }[]
+}
+
+export const bulkCreateDeleteLevelsWorkflowId =
+  "bulk-create-delete-levels-workflow"
+export const bulkCreateDeleteLevelsWorkflow = createWorkflow(
+  bulkCreateDeleteLevelsWorkflowId,
+  (input: WorkflowData<WorkflowInput>): WorkflowData<InventoryLevelDTO[]> => {
+    deleteInventoryLevelsFromItemAndLocationsStep(input.deletes)
```
Comment (carlos-r-l-rodrigues) [inline packages/core-flows/src/inventory/workflows/bulk-create-delete-levels.ts]:
```suggestion
    const deleted = deleteInventoryLevelsFromItemAndLocationsStep(input.deletes)
    removeRemoteLinkStep(deleted)
```

File: packages/medusa/src/api-v2/admin/inventory-items/middlewares.ts [hunk 3]
```diff
@@ -14,6 +15,7 @@ import {
 import { transformBody, transformQuery } from "../../../api/middlewares"
 
 import { MiddlewareRoute } from "../../../types/middlewares"
+import { NextFunction } from "express"
```
Comment (carlos-r-l-rodrigues) [inline packages/medusa/src/api-v2/admin/inventory-items/middlewares.ts]: ?

**Review by carlos-r-l-rodrigues — APPROVED** [review]
💪🏻

File: packages/medusa/src/api-v2/admin/inventory-items/middlewares.ts (hunk 3 above)
Comment (pKorsholm) [reply packages/medusa/src/api-v2/admin/inventory-items/middlewares.ts]: 😅

**Pull request merged**